RecoLocalTracker/SiPixelRecHits/src/SiPixelTemplate.cc

```diff
@@ -2573,6 +2572,7 @@ int SiPixelTemplate::qbin(int id, float cotalpha, float cotbeta, float locBz, fl
    
    
    
+   ilow = ihigh = 0;
```

Comment:
This line is not needed (dead initialization): both ilow and ihigh are re-defined right below

Reply:
OK, I might as well fix it now, since the whole file goes back to Morris anyway due to other changes. (Otherwise, I'd rather he fix it instead :)